### PR TITLE
Add DNS flood job

### LINKED ISSUE #8: DNS stress
- could integrate https://github.com/MickaelBergem/dnsstresss/blob/master/dnsstresss.go

### PR DESCRIPTION
**Pull request by AndriiChuzhynov**
This is first iteration of DNS flood job.
Based on dnsstresss.go but adapted to work with db1000n.
Each job can be pointed on the resolver with multiple target domains, check testconfig.json
May close #8

### CODE REVIEW

File: dnsflood/dnsflood.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	setDefaults(config)
+	s := &DnsStress{
+		Logger: logger,
+		Config: config,
+	}
+
+	parsedResolver, err := ParseIPPort(config.Resolver)
+	config.Resolver = parsedResolver
+	if err != nil {
+		s.Logger.Error("Unable to parse the resolver address", err)
```
Comment (sivillakonski) [inline dnsflood/dnsflood.go:46]: As a hint, you can add the address of the resolver for faster debug

**Comment by AndriiChuzhynov** [comment]
Random subdomain may be a good idea https://github.com/galprz/dns-random-subdomains-ddos-attack

**Comment by Arriven** [comment]
@AndriiChuzhynov I went ahead with #166 instead, although I have a feeling that we can have both jobs even if they're doing similar things. We can always remove one of them later once we test both enough but it's up to you if you want to continue with this PR

**Comment by AndriiChuzhynov** [comment]
Hi @Arriven I'm ok with #166 but, I think, we need to add dynamic nameserver resolving. 
Because NS servers IPs could change pretty fast this days

**Comment by Arriven** [comment]
Can you make an issue or PR for that?

**Comment by AndriiChuzhynov** [comment]
closing in favor of #166

**Pull request closed without merging**